homeassistant/components/whirlpool/strings.json

```diff
@@ -2,10 +2,20 @@
   "config": {
     "step": {
       "user": {
+        "title": "Configure your Whirlpool account",
+        "description": "For 'brand', please choose the brand of the mobile app you use, or the brand of the appliances in your account",
```

Comment:
We have the `data_description` item to describe data items in config flow step forms.

Example:
https://github.com/home-assistant/core/blob/38164c705e7faa3b87fc2117b2e4dd8884aff8b5/homeassistant/components/adguard/strings.json#L14-L16

Reply:
Thanks for pointing that out, I missed that my first time through the docs. I've switched the brand info to the data description now